rts/Lua/LuaVBOImpl.cpp

```diff
@@ -1004,6 +1121,15 @@ size_t LuaVBOImpl::UploadImpl(const std::vector<TIn>& dataVec, uint32_t elemOffs
 	return uploadToGPU(bytesWritten);
 }
 
+
+/*** Binds engine side vertex or index VBO containing models (units, features) data.
+ *
+ * @function VBO:ModelsVBO
+ *
+ * Also fills in VBO definition data as they're set for engine models (no need to do VBO:Define()).
+ *
+ * @treturn nil|number size
```

Comment:
Most likely not `size`, I was not able to parse what kind of types it returns. (generally returning `size_t` means multireturn)

Reply:
I think it's buffer size in bytes actually.

Reply:
`return bufferSizeInBytes;` So I assume it returns engine's VBO buffer size, not the most useful thing to return, but it's what it is.